drivers/overlay/encryption.go

```diff
@@ -31,6 +31,12 @@ const (
 	bidir
 )
 
+var (
+	saTxHardLimit = netlink.XfrmStateLimits{TimeHard: uint64(12*3600 + 60)}
+	saRxHardLimit = netlink.XfrmStateLimits{TimeHard: uint64(3*12*3600 + 60)}
```

Comment:
Will these hardlimits kick in if there are no traffic on those encrypted tunnels ? 
If yes, and if the hardlimit kicks in, how will the entries be reestablished ? 
(in other words, is it true that the entries will be removed only if the daemon is down - as mentioned in the PR description ? )